website/docs/contributing/writing-guide.md

```diff
@@ -111,7 +111,22 @@ It can also be used to highlight specific texts, including URLs, file names, and
 
 Always specify the language in Markdown code blocks.
 
-For MongoDB shell commands, use `js` language.
+### MongoDB shell commands and results
+
+#### Documentation
+
+For our documentation, we use the CTS tool to test and validate the code snippets for the MongoDB shell commands and responses.
+All MongoDB shell commands and responses should be written in the `website/docs/guides/` folder in extended JSON format.
```

Comment:
Even if the files are used in `website/docs/reference` like in this `website/docs/reference/administrative-commands/createIndexes.mdx` right? I understand it's best to keep cts all in one place.

Reply:
Yes, all CTS will be under the guides section regardless of where it's linked from.

Reply:
But… why? That doesn't make sense

Reply:
We wanted to have a CTS data set under guides for guides only. Now, they do not even share the fixture (#5131). Of course, we should not have those files for reference documentation under /guides/